src/content/docs/network-performance-monitoring/setup-performance-monitoring/network-flow-monitoring.mdx

```diff
@@ -368,13 +368,13 @@ When planning your strategy for collecting network flows at scale, New Relic rec
 ## Set up network flow data monitoring in New Relic [#setup-network-flow-monitoring]
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Add more data**.
-2. Scroll down until you see **Network monitoring** and click **Network Flows**.
-3. Follow the steps in New Relic.
+2. Scroll down until you see **Network** and click **Network Flows**.
+3. Follow the steps outlined in the New Relic UI.  The available installation methods are a Docker or Linux package manager.
```

Comment:
Can this be changed to - "The available installation methods are Docker or Linux package manager."  Removing the "a".